lib/ansible/modules/cloud/vmware/vmware_guest_disk_info.py

```diff
@@ -230,6 +230,7 @@ def gather_disk_info(self, vm_obj):
                     disks_info[disk_index]['backing_writethrough'] = bool(disk.backing.writeThrough)
                     disks_info[disk_index]['backing_thinprovisioned'] = bool(disk.backing.thinProvisioned)
                     disks_info[disk_index]['backing_eagerlyscrub'] = bool(disk.backing.eagerlyScrub)
+                    disks_info[disk_index]['backing_diskmode'] = disk.backing.diskMode
```

Comment:
```suggestion
                    disks_info[disk_index]['backing_diskmode'] = disk.backing.diskMode
                    disks_info[disk_index]['backing_disk_mode'] = disk.backing.diskMode
```

For backward compatability.

Reply:
This change is required due to https://github.com/ansible/ansible/commit/34868f07d5b32083d03cc4954287355b3b2f647e#commitcomment-35271563